immersion/kubeflow_pipelines/cicd/labs/lab-03_vertex.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<p>nit: Can we hyperlink the repo: https://github.com/GoogleCloudPlatform/mlops-on-gcp? Since the repo is usually already clones in their notebook they don't know which repo to fork</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Good catch! Thanks. Done </p>